### PR TITLE
Add bigdl-llm-finetune-cpu to manually_build to upload image on hub

### PR DESCRIPTION
**Pull request by Jasonzzt**
changed the files .github/workflows/manually_build.yml & .github/workflows/manually_build_for_testing.yml

### CODE REVIEW

**Comment by Jasonzzt** [comment]
The first two commits make no sense, please ignore them.

**Review by liu-shaojun — APPROVED** [review]
LGTM

File: .github/workflows/manually_build.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ on:
         type: choice
         options:
         - all
+        - bigdl-llm-finetune-cpu
```
Comment (glorysdj) [inline .github/workflows/manually_build.yml:13]: bigdl-llm-finetune-lora-cpu
and
bigdl-llm-finetune-qlora-xpu

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**